### DISCUSSION TITLE
Hepatic impairment population

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1341] [Questions & Problems] [answered]
Hello everyone.
Based on the scaling factors in Edgiton et al 2008, I tried to create a population with cirrhosis, but the results were inconsistent with the OSPS tutorial. For example, in the tutorial file: Bone blood flow rate control: 2.75 mL/min/100g organ; Child-Pugh A: 4.42 mL/min/100g organ. This does not seem to match the 1.75 folds in the Edginton et al 2008.
**I am wondering how should I modify the blood flow? Is it to modify the special blood flow ratio?**
In addition, what parameters should **Cardiac index, portal blood flow, and hepatic arterial** correspond to? Is the portal blood flow ratio? (This seems to be automatically calculated in PK-Sim.) The liver specific blood flow ratio?
In addition, why is the special blood flow ratio of the **small intestine, spleen and stomach** lower? (The control group is 2.5 times the Child-Pugh A group?) Why not have a higher trend like other organs?
![image](https://user-images.githubusercontent.com/86552551/196213795-c07ad8db-7239-4cad-814a-1eed532037f1.png)
![image](https://user-images.githubusercontent.com/86552551/196213946-c916cb87-5fe8-4b52-9ac5-e17cfdca63dd.png)
![image](https://user-images.githubusercontent.com/86552551/196214087-7638148e-7de8-437c-a117-0b849d066dcb.png)
![image](https://user-images.githubusercontent.com/86552551/196214194-43c2d937-0ef7-43ce-bf3f-38c9c6c9a1ca.png)
![image](https://user-images.githubusercontent.com/86552551/196214354-aaeabe55-2e5a-4a94-b7cf-9f6f937224e5.png)

**Comment by tobiasK2001** [comment]
hm, probably a version issue. Can you please specify which tutorial file you refer to?
Best, Tobias

**Reply by SCZ1174** [reply]
Thank you very much for your reply, the version of the tutorial file is OSPS Version 11.
![image](https://user-images.githubusercontent.com/86552551/196626521-e7aa9b57-6ef6-4b98-9b06-f21d5ade676e.png)

**Comment by tobiasK2001** [comment]
Ah Ok. In that case I am sure someone at ESQlab can sort that out. Isn't that so Stephan (@stephan.schaller@esqlabs.com)?

Best, Tobias

**Comment by StephanSchaller** [comment]
Dear @SCZ1174 , the exercise is an adoption of the work by Edginton et al 2008 as re-worked as a published exercise with the [PBPK book from SA Peters](https://www.wiley.com/en-us/Physiologically+Based+Pharmacokinetic+(PBPK)+Modeling+and+Simulations:+Principles,+Methods,+and+Applications+in+the+Pharmaceutical+Industry,+2nd+Edition-p-9781119497684). 
Indeed, as stated in the exercise, modify the "Specific blood flow rate":
![image](https://user-images.githubusercontent.com/25477643/196709677-f337147e-1958-457f-a2f8-3179149fa85b.png)

The other parameters you list are parameters in PK-Sim. Please elaborate on your corresponding questions as I am not sure I understand what answer you are looking for exactly.

Thanks,
Stephan

**Reply by SCZ1174** [reply]
Thanks for your reply, I have listed my questions:
1. For blood flow parameters, how can I scale? Is it directly multiplied by the corresponding proportional parameter based on the control group? But it seems to be inconsistent with the tutorial file. (For example, bones should belong to "other organs" The ratio is A: 1.75; B: 2.25; C: 2.75. But the modified value in the tutorial file is control: 2.75 ml/min/100g organ A: 4.42 ml/min/100g organ B: 5.47 ml/min/100g organ C: 6.28 ml/min/100g organ. If scaling is used directly, it should be: Control: 2.75 ml/min/100g organ A: 4.8125 ml/min/100g organ B: 6.1875 ml/min/100g organ C: 7.5625 ml/min/100g organ.)

2. Which parameter should be modified of Cardiac index?

3. What parameters do portal and hepatic arterial correspond to? Is it portal vein and Liver blood flow?

4. Why is the special blood flow ratio of the small intestine, spleen and stomach lower? (The control group is 2.5 times the Child-Pugh A group?) Why not have a higher trend like other organs? (like the proportion of "other organs")

**Comment by AnnikaRPS** [comment, marked as answer]
Hi @SCZ1174 
The change in the portal blood flow rate refers to the blood flow rates of all portal vein supplying organs (large intestine, pancreas, small intestine, spleen, stomach) as the portal vein blood flow rate in PKSim is implemented as the sum of these organ blood flow rates. The change in hepatic arterial blood flow rate refers to the blood flow rate of the liver in PKSim. Note that the specific liver blood flow is calculate from two different factors. Firstly, it is changed according to the values from Edginton 2008.  Secondly, the liver volume is changed. As the total blood flow is calculated from the specific blood flow and the organ volume, the specific blood flow has to be scaled, so that the changed organ volume does not have an impact on the total blood flow (as the total blood flow was already adapted to liver cirrhosis).
As described by Edginton and Willman in the section “Physiological Scaling: Blood Flows” the changes in liver blood flow, portal venous blood flow and renal blood flow were informed from literature values. Additionally, the change in cardiac index is informed from literature. As there is no direct representation of the cardiac index in PKSim, its change was used to calculate the change in blood flow rates of “other organs”. The same was done in the tutorial. So instead of using the factor on “other organs” from Edginton and Willman we used the factor on cardiac index from the same publication to calculate the change in other organs as only the change in cardiac index is informed by literature and, therefore, more reliable. 
The cardiac index (CI) equals the heart minute volume (HMV) per body surface area (BSA). The heart minute volume is the sum of all blood flows of the body circulation (excluding the pulmonary blood flow), with Qrest being the sum of the portal, hepatic arterial and renal blood flow and Qother being all other organs (bone, fat, gonads, heart, muscle, skin). As the brain blood flow is maintained stable in cirrhosis, it will not be included in Qother in the equations.
𝐶𝐼 = 𝐻𝑀𝑉/𝐵𝑆𝐴 = (𝑄𝑜𝑡ℎ𝑒𝑟 + 𝑄𝑟𝑒𝑠𝑡 + Qbrain)/𝐵𝑆𝐴
In the disease state the CI and Qrest change to a known degree (DF_CI and DF_rest; “DF” stands for “disease factor”) and the body surface area stays the same. 
𝐷𝐹_𝐶𝐼 = 𝐶𝐼𝑑𝑖𝑠𝑒𝑎𝑠𝑒/𝐶𝐼 = (𝑄𝑜𝑡ℎ𝑒𝑟 × 𝐷𝐹_𝑜𝑡ℎ𝑒𝑟 + 𝑄𝑟𝑒𝑠𝑡 × 𝐷𝐹_𝑟𝑒𝑠𝑡+ Qbrain)/(𝑄𝑜𝑡ℎ𝑒𝑟 + 𝑄𝑟𝑒𝑠𝑡+ Qbrain)

The calculation using the number from the tutorial PK-Sim files (for Child-Pugh A):
DF_CI = 1.11
Qrest [L/min] = 1.21 + 0.29 + 1.33 = 2.83 (sum of healthy values of portal, hepatic arterial and renal blood flow in the individual that will be transformed to Child-Pugh A)
Qother [L/min] = 0.32 + 0.32 + 0.00325 + 0.26 + 1.12 +  0.33 = 2.35325 (sum of healthy values of “other organs” in the individual that will be transformed to Child-Pugh A)
Qbrain  [L/min] = 0.78
(𝑄𝑟𝑒𝑠𝑡 × 𝐷𝐹_𝑟𝑒𝑠𝑡) [L/min] = 1.21 * 0.4 + 0.43 * 1.3 + 1.33 * 0.88 = 2.2134 (sum of Child-Pugh A adapted values of portal, hepatic arterial and renal blood flow)
Put all the numbers into the equation above:
1.11=  (2.35325 × 𝐷𝐹_𝑜𝑡ℎ𝑒𝑟 + 2.2134 + 0.78)/( 2.35325 + 2.83 + 0.78)
<-> 𝐷𝐹_𝑜𝑡ℎ𝑒𝑟  = (1.11*( 2.35325 + 2.83 + 0.78) -2.2134 - 0.78)/ 2.35325 = 1.54
So the factor applied to “other organs” is ~1.54 (might be a little different in the tutorial due to rounding etc.).
Hope that helps.

**Reply by StephanSchaller** [reply]
Thanks so much for these clarifications, Annika. I did not yet find the time to answer :-)

**Reply by SCZ1174** [reply]
Thanks a lot for your help, it worked great for me!

Best regards,
SCZ

**Reply by msevestre** [reply]
FYI: The upcoming HI implementation in PK-Sim will use the very same approach.
